packages/webex/src/calling.js

```diff
@@ -137,8 +137,8 @@ class Calling extends EventEmitter {
     return WebexCalling.createMicrophoneStream;
   }
 
-  static createNoiseReductionEffect(authToken) {
-    return new WebexCalling.NoiseReductionEffect({authToken});
+  static createNoiseReductionEffect(options) {
```

Comment:
Should we make this change backward compatible? Isn't this a breaking change for developers?

Reply:
We will update the documentation as this is a better solution